synapse/api/constants.py

```diff
@@ -196,9 +196,6 @@ class EventContentFields:
     MSC2716_CHUNK_ID = "org.matrix.msc2716.chunk_id"
     # For "marker" events
     MSC2716_MARKER_INSERTION = "org.matrix.msc2716.marker.insertion"
-    MSC2716_MARKER_INSERTION_PREV_EVENTS = (
-        "org.matrix.msc2716.marker.insertion_prev_events"
-    )
```

Comment:
I don't think we will use this 🤷

Reply:
Switched to backfilling the insertion event in https://github.com/matrix-org/synapse/pull/10420